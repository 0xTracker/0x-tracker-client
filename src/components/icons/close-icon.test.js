import { render } from 'react-testing-library';
import React from 'react';

import CloseIcon from './close-icon';

describe('close icon component', () => {
  it('should render without props', () => {
    const { container } = render(<CloseIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
