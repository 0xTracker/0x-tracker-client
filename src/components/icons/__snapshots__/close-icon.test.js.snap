// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`close icon component should render without props 1`] = `
.c0 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

<svg
  aria-hidden="true"
  class="c0"
  fill="currentColor"
  focusable="false"
  viewBox="0 0 24 24"
>
  <path
    d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
  />
</svg>
`;
